plugins/UsersManager/tests/System/expected/test___UsersManager.getUser_login1_when_superuseraccess.xml

```diff
@@ -2,7 +2,7 @@
 <result>
 	<row>
 		<login>login1</login>
-		<password>5f4dcc3b5aa765d61d8327deb882cf99</password>
+		<password>$2y$10$/Lz0vEwEOnFNt17Gak7Js./9OJXOKN7lT5ObczOTBNtmU/ATvoGfe</password>
```

Comment:
FYI: In `plugins/UsersManager/tests/System/ApiTest.php:42` we can add `password` to `xmlFieldsToRemove` array and then it won't appear anymore and they won't fail randomly